config/locales/nl.yml

```diff
@@ -220,7 +220,7 @@ nl:
   applications:
     invalid_url: De opgegeven URL is ongeldig
   auth:
-    agreement_html: Wanneer je op registeren klikt ga je akkoord met <a href="%{rules_path}">onze gebruikersvoorwaarden</a> en <a href="%{terms_path}">ons privacybeleid</a>.
```

Comment:
'Another people's opinion' for a (self-made) typo? 

'registeren' is not a word. 'Registreren' is. It means in English 'to register' (an account). I asked @Gargron to come here, cause I find this not very productive.